### DISCUSSION TITLE
react-hook-form website not accessible？

### DISCUSSION THREAD

**Discussion by Cyberhan123** [react-hook-form/react-hook-form#5719] [Q&A] [answered]
the chrome report this error: ERR_SSL_PROTOCOL_ERROR, I reset my chrome and use postman send an get request,but it's also have same problem...
![image](https://user-images.githubusercontent.com/38837039/123215698-59003e00-d4fb-11eb-8115-72ad2528fdcf.png)
anyone have same problem?

**Comment by jorisre** [comment, marked as answer]
It works for me, do you still have the issue?

Which OS and browser are you using?

**Reply by Cyberhan123** [reply]
thanks,I haven't report an issue.
os: Windows 10 20H2
browser:Chrome 91.0.4472.106

**Reply by NorviNS** [reply]
os: Mac 10.15
browser: Chrome 91

I have the similar issue... guessing the website not available in China.
![image](https://user-images.githubusercontent.com/6480233/126426034-d6167340-eccc-411c-a36c-c5355c4d6b13.png)

**Reply by bluebill1049** [reply]
really?? no wonder we have very low traffic from China. can you double check?

**Reply by NorviNS** [reply]
[No access to official website in China?](https://github.com/react-hook-form/react-hook-form/discussions/5261)
I really can't access to the website, and seems like not only me.

**Reply by bluebill1049** [reply]
yes... confirmed: https://twitter.com/_egoistlily/status/1393199775147692033

**Reply by bluebill1049** [reply]
we are getting amazing help from Vercel. should be able to resolve this one.

**Reply by NorviNS** [reply]
thank you~

**Reply by bluebill1049** [reply]
can you confirm it's working?

**Reply by NorviNS** [reply]
yes! it's working!
![F3D25FB4-07D5-4A70-B699-A9B16A65D5BB](https://user-images.githubusercontent.com/6480233/126482503-c78a784c-e551-4f5a-aa2e-c4f1a5ff0294.png)

**Reply by bluebill1049** [reply]
amazing!
